OpenRA.Mods.Cnc/UtilityCommands/LegacyTilesetImporter.cs

```diff
@@ -146,7 +146,9 @@ void IUtilityCommand.Run(Utility utility, string[] args)
 
 								data.AppendLine($"\t\t\t\tMinColor: {s.ReadUInt8():X2}{s.ReadUInt8():X2}{s.ReadUInt8():X2}");
 								data.AppendLine($"\t\t\t\tMaxColor: {s.ReadUInt8():X2}{s.ReadUInt8():X2}{s.ReadUInt8():X2}");
-								data.AppendLine($"\t\t\t\tZOffset: {(-tileSize.Height / 2.0f)}");
+								#pragma warning disable SA1021
+								data.AppendLine($"\t\t\t\tZOffset: {-tileSize.Height / 2.0f}");
+								#pragma warning restore SA1021
```

Comment:
What here warrants the disabling?

Reply:
`-` wants a space in front, but when add it then the `{` says it doesn't want a space after it